src/vrapp/VRVolumeApp.cpp

```diff
@@ -622,26 +597,18 @@ void VRVolumeApp::render(const MinVR::VRGraphicsState& renderState)
       m_models_MV[i] = volume_mv;
 
       /*Original transformations of the model - DO NOT REMOVE YET*/
```

Comment:
add a `TODO` with the issue number for this

Reply:
I removed those comments. No longer needed them